### PR TITLE
added necessary packages for apollo and graphql, connected our app to…

### PR DESCRIPTION
**Pull request by jenkens-dev**
… the apollo client, queried the graphql server for data instead of using mock data

### CODE REVIEW

**Comment by jenkens-dev** [comment]
@Alphy11

File: src/components/EventCardCollection.js [hunk 1]
```diff
@@ -1,38 +1,63 @@
...
+`;
+
+const EventCardCollection = () => {
+  //useQuery hook takes our gql query and makes a request to the backend
+  //the return value is held in data
+   const { loading, error, data } = useQuery(GET_THREE_EVENTS);
+   if (loading) return null;
+   return (
+      <div>
+         {data.event.length !== 0 ? (
```
Comment (Alphy11) [inline src/components/EventCardCollection.js]: Shouldn't this be if there is an error?

File: src/index.tsx [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+//the client represents the endpoint to our graphql server
+
+const client = new ApolloClient({
+   uri: 'https://play-it-forward-graphql.herokuapp.com/v1/graphql',
+});
+
+//ApolloProvider allows us to access the client from anywhere in the component tree
+
+ReactDOM.render(
+   <ApolloProvider client={client}>
```
Comment (Alphy11) [inline src/index.tsx]: Add this into your App, not here.

File: src/App.tsx [hunk 3]
```diff
@@ -1,11 +1,11 @@
 // React Imports
```
Comment (Alphy11) [inline src/App.tsx:1]: you can set up this thing called husky that will let you add event hooks (not react hooks) to your repo, so you can run a  script before committing, https://dev.to/botreetechnologies/setting-up-husky-pre-commit-hook-with-eslint-prettier-and-lint-staged-for-react-and-react-native-d05

**Review by Alphy11 — APPROVED** [review]
@Taljjaa No major comments

**Pull request merged**